engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/UpdateByOperatorFactory.java

```diff
@@ -1371,52 +1377,84 @@ private UpdateByOperator makeRollingFormulaOperator(@NotNull final MatchPair pai
             final long prevWindowScaleUnits = rs.revWindowScale().getTimeScaleUnits();
             final long fwdWindowScaleUnits = rs.fwdWindowScale().getTimeScaleUnits();
 
+            final String formula = rs.formula();
```

Comment:
Maybe centralize param token access and validation, too?
```suggestion
            final String formula = rs.formula();
            //noinspection deprecation
            final String paramToken = rs.paramToken().orElseThrow();
```